homeassistant/components/shelly/sensor.py

```diff
@@ -961,6 +961,12 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         device_class=SensorDeviceClass.BATTERY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "analoginput_xpercent": RpcSensorDescription(
+        key="input",
+        sub_key="xpercent",
+        name="Analog value",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
I'm not sure a state class is too useful without a UOM?

Reply:
I didn't find anything in the documentation that would suggest whether we should drop `state_class` in this case. Do you think we should drop it?

Reply:
If we ever did add a UOM we would end up here

https://github.com/home-assistant/core/blob/46f8fb3ac19ca9810a3399520b59a119a6736614/homeassistant/components/sensor/recorder.py#L237

Unless you have a strong need to long term stats for this, it seems like it would be better to wait until we have a UOM to add it so we don't add stats into the database with a `None` UOM.

Reply:
Good point, I'll remove `state_class`.